src/Platform.Handlers/src/Xamarin.Platform.Handlers/Primitives/CornerRadius.cs

```diff
@@ -3,7 +3,6 @@
 namespace Xamarin.Forms
 {
 	[DebuggerDisplay("TopLeft={TopLeft}, TopRight={TopRight}, BottomLeft={BottomLeft}, BottomRight={BottomRight}")]
-	[TypeConverter(typeof(CornerRadiusTypeConverter))]
 	public struct CornerRadius
```

Comment:
It did break Forms... We probably need to wrap this with `#if` for now?

Reply:
I could attach this attribute to all properties that I find, but it still will break anyone who creates a new control and uses this type. This really needs to be on the struct type so it can be consumed. Or, Forms needs a way to register things like this.

Reply:
I might be able to get it to work by adding this to the `TypeConverterAttribute.KnownConverters` list...

Reply:
This should be fixed in this: https://github.com/xamarin/Xamarin.Forms/pull/12529/commits/39caa92837da445debeb258cf07f8bbff6bbf986

CI will confim.